### ISSUE TITLE
Fixed header when switching tabs 

### ISSUE THREAD

**Issue by isamuhiro** [aksonov/react-native-router-flux#3007]
### Version
Tell us which versions you are using: 

- "react-native-router-flux": "^4.0.0-beta.28",
- "react-native": "~0.54.1",

What would be a good approach to create a header which can be fixed in all tabbar/stacked childrens?
Can you guys provide a snack example or some workaround?

**Comment by telkandrew** [comment]
K

**Comment by telkandrew** [comment]
Klkmkk

**Comment by daviscabral** [comment]
@isamuhiro could you find a solution for this?

**Comment by isamuhiro** [comment]
@daviscabral yeah, I just declared an Header component into the navBar props on <Stack/> component. It should be the pattern component of <Tabs/>. 
Should be something like this:
`<Stack key="app"  navBar={Header}>
  <Tabs>
//Your scene components containing a fixed header 
</Tabs>
</Stack>
`

**Comment by daviscabral** [comment]
Great - thank you for taking the time to write about it here.
